src/audiounit.cs

```diff
@@ -437,7 +486,7 @@ interface AUParameter : NSSecureCoding
 		[Export ("setValue:originator:atHostTime:eventType:")]
 		void SetValue (float value, IntPtr originator, ulong hostTime, AUParameterAutomationEventType eventType);
 
-		[iOS (10,0), Mac (10,12, onlyOn64 : true)]
+		[iOS (10,0), Mac (10,12, onlyOn64 : true), Watch (4, 0), TV (11, 0)]
 		[TV (10,0)]
```

Comment:
it's decorated with both `[TV(10,0]` and `[TV(11,0)]` ?
please check back if Xcode8 has this for tvOS